app/src/main/java/de/westnordost/streetcomplete/map/MapFragment.kt

```diff
@@ -193,7 +193,13 @@ open class MapFragment : Fragment(),
     }
 
     protected open suspend fun getSceneUpdates(): List<SceneUpdate> {
-        return listOf(SceneUpdate("global.language", Locale.getDefault().language))
+        val textScale = getResources().getConfiguration().fontScale;
```

Comment:
Don't you get suggestions/soft warnings here in Android Studio?

Reply:
No, without any protests. Is there anything wrong/imperfect here?

Reply:
I reset Android Studio suggestion settings (I disabled some due to completely spurious reports) and it appeared as expected.